sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducerAsyncClient.java

```diff
@@ -115,8 +116,8 @@
      * load balance the messages amongst available partitions.
      */
     EventHubProducerAsyncClient(String fullyQualifiedNamespace, String eventHubName, EventHubConnection connection,
-        AmqpRetryOptions retryOptions, TracerProvider tracerProvider, MessageSerializer messageSerializer,
-        boolean isSharedConnection) {
+                                AmqpRetryOptions retryOptions, TracerProvider tracerProvider,
```

Comment:
revert indentation. if the params are squished against the right margin, it should have an 8 space indentation.